ios/RCTMapboxGL/RCTMapboxGL.h

```diff
@@ -33,8 +33,9 @@
 - (void)setDirectionAnimated:(int)heading;
 - (void)setCenterCoordinateAnimated:(CLLocationCoordinate2D)coordinates;
 - (void)setCenterCoordinateZoomLevelAnimated:(CLLocationCoordinate2D)coordinates zoomLevel:(double)zoomLevel;
-- (void)selectAnnotationAnimated:(NSUInteger)annotationInArray;
-- (void)removeAnnotation:(NSUInteger)annotationInArray;
+- (void)selectAnnotationAnimated:(NSString*)selectedId;
```

Comment:
The prevailing style in Objective-C is to capitalize “ID”, or better yet, to spell it out as “identifier”.